spectests/structs_test.go

```diff
@@ -22,37 +22,46 @@ type codec interface {
 	ssz.Marshaler
 	ssz.Unmarshaler
 	ssz.HashRoot
+
+	// TODO
+	GetTreeWithWrapper(w *ssz.Wrapper) (err error)
```

Comment:
Create a custom interface since not all the structs right now can implement the tree
```
type codecTree interface {
    GetTreeWithWrapper(w *ssz.Wrapper) (err error)
}
```